src/rpc/misc.cpp

```diff
@@ -44,7 +44,7 @@ UniValue getinfo(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 0)
         throw runtime_error(
             "getinfo\n"
-            "Returns an object containing various state info.\n"
+            "\nDEPRECATED. Returns an object containing various state info.\n"
```

Comment:
I wonder if anyone types in `help getinfo`. (Otherwise no one will see the DEPRECATED warning anyway)